systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -979,6 +979,13 @@ public static void createFileSinkConnector(String podName, String topicName) {
         );
     }
 
+    public static void waitUntilKafkaConnectServiceIsPresent(String podNamePrefix) {
+        LOGGER.info("Waiting until kafka connect service is present");
+        TestUtils.waitFor("Waiting until kafka connect service is present", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_STATUS_TIMEOUT,
+            () -> cmdKubeClient().execInPod(podNamePrefix, "/bin/bash", "-c", "curl http://localhost:8083/connectors").out().contains("[]\n"));
```

Comment:
because it's just a check to verify that the Kafka Connect REST is up and running, we could use `curl -I`. It sends a HEAD request instead of GET and you can check on the 200 status code instead of an empty array [ ].
Not sure if the HTTP REST interface in Kafka Connect replies to HEAD request. Maybe you can try?

Reply:
Nice suggestion. Thanks @ppatierno.